src/modules/form/components/viewable/DynamicViewItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Grid } from '@mui/material';
```

Comment:
This new component is extracted out from the deleted code in DynamicViewFields.tsx, to avoid repeated code - but maybe it's not worth it

Reply:
Thank you for taking initiative on this one @martha ! Hmm, I think I prefer not to add a new factored-out component (`DynamicViewItem`). There wouldn't be much repeated code if we do it like this in `DynamicFormFields.tsx`:

```
    const itemComponent = item.readOnly ? (
      <DynamicViewField
        item={item}
        value={values[item.linkId]}
        disabled={isDisabled}
        nestingLevel={nestingLevel}
        horizontal={horizontal}
        pickListArgs={pickListArgs}
        // Needed because there are some enable/disabled and autofill dependencies that depend on PickListOption.labels that are fetched (PriorLivingSituation is an example)
        adjustValue={itemChanged}
        {...props}
      />
    ) : (
      // render DynamicField in the same way we do now
    );

  // ...  renderFn code is already present
```

Reply:
I added a quick story to test out how this looks. Can you commit it to `DynamicForm.stories.tsx`?

```
# Render the DynamicForm with values, modified such that every item is marked "readOnly" in the FormDefinition
export const WithValuesAsReadOnly = Template.bind({});
WithValuesAsReadOnly.args = {
  definition: modifyFormDefinition(
    formDefinition,
    (item) => (item.readOnly = true)
  ),
  initialValues: ViewStory.args?.values,
  errors: emptyErrorState,
};

```